<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabnd</class>
 <widget class="QWidget" name="tabnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>646</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>File preview:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
     <item>
      <widget class="QTextEdit" name="nd_previewedit">
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="ndoptionsgrp">
         <property name="title">
          <string>Options</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Shape</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="HistoryCombo" name="nd_shapeedit">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of numerical entries (separated by space or comma), giving the dimensions of each axis. Use -1 to automatically detect axis length (only one axis can be -1). Auto means autodetect shape, based on number of blank lines.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="HistoryValueCombo" name="nd_csvlocale">
            <property name="toolTip">
             <string>Numerical format of numbers in file:
 System - what this computer is set to use
 English - format 123,456.78
 European - format 123.456,78</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CSV text delimeter</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="HistoryValueCombo" name="nd_mode">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use text mode, with space-separated values or CSV mode. When using CSV mode, the delimiters and numeric locale are given below.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>CSV delimeter</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>CSV locale</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="HistoryCombo" name="nd_csvdelim">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="HistoryCombo" name="nd_csvtextdelim">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="HistoryCheck" name="nd_transposecheck">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Transpose dimensions. Normally data are in 'C' order, with last axis changing fastest. Setting this will change to 'Fortran' order, with 1st axis changing fastest.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Trans&amp;pose</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Oxygen-Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;File should consist of either:&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1. A set of numbers, in free-form style, and the shape specified above or using the command &amp;quot;shape A B Câ€¦&amp;quot; in the file. Shapes should be positive integers, though one dimension can be -1 to automatically detect the length.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;2. A set of numbers arranged in rows and columns. 1D datasets are on a single row, 2D datasets are a set of 1D rows, 3D as a set of 2D datasets separated by blank rows, 4D as a set of 3D datasets separated by two blank rows, etc.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Data are ordered in C format, with the rightmost index increasing most rapidly. The transpose option can be used to change the option. This can also be included in the file with a single row containing &amp;quot;transpose&amp;quot;.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Dataset:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="HistoryCombo" name="nd_datasetedit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dataset name to import&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistoryCombo</class>
   <extends>QComboBox</extends>
   <header>historycombo.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryCheck</class>
   <extends>QCheckBox</extends>
   <header>historycheck.h</header>
  </customwidget>
  <customwidget>
   <class>HistoryValueCombo</class>
   <extends>QComboBox</extends>
   <header>historyvaluecombo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
